<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hello_fresh_student</class>
 <widget class="QDialog" name="hello_fresh_student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/school_logo/C:/Users/Defend/Desktop/picture/tree.jpg);</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>521</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/school_logo/C:/Users/Defend/Desktop/picture/QQ1.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>20</width>
     <height>391</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/school_logo/C:/Users/Defend/Desktop/picture/download.jpg)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>701</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/school_logo/C:/Users/Defend/Desktop/picture/school_2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="map_baidu_label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>430</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/school_logo/C:/Users/Defend/Desktop/picture/map_baidu.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="map_cqut_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>551</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/school_logo/C:/Users/Defend/Desktop/picture/map_cqut.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="map_cqut_title_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>XXXX大学位置图</string>
   </property>
  </widget>
  <widget class="QLabel" name="map_baidu_title_label">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>380</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>XXX校区百度地图导航二维码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/school_logo/C:/Users/Defend/Desktop/picture/quit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ff5500;&quot;&gt;欢迎：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="student_name_label">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>41</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>41</y>
     <width>30</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>同学</string>
   </property>
  </widget>
  <widget class="QPushButton" name="school_local_pushButton">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>193</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>地理位置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="school_info_pushButton">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>111</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>国院简介</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fun_pushButton">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>357</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>周边美食</string>
   </property>
  </widget>
  <widget class="QPushButton" name="collage_pushButton">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>275</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>信息录入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>校区游览</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
